eahub/feedback/apps.py

```diff
@@ -0,0 +1,6 @@
+from django import apps
+
+
+class FeedbackConfig(apps.AppConfig):
```

Comment:
I was getting 
>>> Model class eahub.feedback.models.Feedback doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.  

when running locally, so I've added  the feedbacks app to installed apps